src/combinatorics/simple_graph/connectivity.lean

```diff
@@ -1401,6 +1401,19 @@ protected lemma reachable.map {G : simple_graph V} {G' : simple_graph V'}
   (f : G →g G') {u v : V} (h : G.reachable u v) : G'.reachable (f u) (f v) :=
 h.elim (λ p, ⟨p.map f⟩)
 
+lemma iso.reachable_iff {G : simple_graph V} {G' : simple_graph V'}
+  {φ : G ≃g G'} {u v : V} : G.reachable u v ↔ G'.reachable (φ u) (φ v) :=
+⟨reachable.map φ.to_hom, λ r, (φ.left_inv u) ▸ (φ.left_inv v) ▸ (r.map φ.symm.to_hom)⟩
+
+lemma iso.reachable_iff' {G : simple_graph V} {G' : simple_graph V'}
```

Comment:
You can look at [`order_iso.symm_apply_le`](https://leanprover-community.github.io/mathlib_docs/order/hom/basic.html#order_iso.symm_apply_le) for a better naming convention.